tests/antares/services/api_services/test_study_api.py

```diff
@@ -472,3 +474,36 @@ def test_read_outputs(self):
             mocker.get(run_url, json={"description": error_message}, status_code=404)
             with pytest.raises(OutputsRetrievalError, match=error_message):
                 self.study.read_outputs()
+
+            # ==== get_matrix() ====
+
+            matrix_url = f"https://antares.com/api/v1/studies/{self.study_id}/raw?path=output/{output1.name}/economy/mc-all/grid/links"
+            matrix_output = {"columns": ["upstream", "downstream"], "data": [["be", "fr"]]}
+            mocker.get(matrix_url, json=matrix_output)
+
+            matrix = output1.get_matrix("mc-all/grid/links")
+            assert isinstance(matrix, pd.DataFrame)
+            assert matrix.shape[1] == len(matrix_output["columns"])
```

Comment:
Can't you just put matrix_output inside a dataframe and test the equality ? It would be clearer
Same remark for the aggregate values part